src/gmt_common_longoptions.h

```diff
@@ -52,12 +66,12 @@
     {   0, 'B', "axis",
                 "x,y,z",                 "x,y,z",
                 "a,f,l,L,p,s,S,u",       "angle,fancy,label,hlabel,prefix,alt_label,alt_hlabel,unit" },
-    {   0, 'J', "projection",
+    {   0, 'J', "projection|proj",
                 "",                      "",
                 "d,a,t,v,w,z,f,k,r",     "d,a,t,v,w,z,f,k,r" },
-    {   0, 'R', "region",
+    {   0, 'R', "region|reg|domain",
```

Comment:
``reg`` clashes with ``registration`` and is not a good alias for -R. Julia uses ``region`` or ``limits`` (like Matlab's xlim, ylim, etc). Why introducing ``domain``?